src/components/BlogPostPreview.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react"
+import { unified } from "unified"
+import remarkParse from "remark-parse"
+import remarkHtml from "remark-html"
+import { useEffect } from "react"
+
+import TitleWrapper from "~/src/components/blog/post/wrapper"
+import Wrapper from "~/src/components/blog/post/wrapper"
+import BodyWrapper from "~/src/components/blog/post/body_wrapper"
+import Body from "~/src/components/blog/post/body"
+import "~/src/common/base.scss"
+import Title from "~/src/components/blog/title"
+
+import * as styles from "~/src/templates/blog/post.module.scss"
+import * as titleStyles from "~/src/components/blog/post/header.module.css"
+
+const BlogPostPreview = (props) => {
+  const md = props.entry.getIn(["data", "body"])
+  const [html, setHTML] = useState("")
+
+  useEffect(() => {
+    ;(async () => {
```

Comment:
Is this semicolon on purpose?

Reply:
Prettier adds it automatically. I think it's to avoid some syntax error because there's a ( after {.